### PR TITLE
show service account status and expiry in ui

### PR DESCRIPTION
**Pull request by prakashsvmx**
show service account status and expiry in ui
Fixes https://github.com/minio/console/issues/2848

How does it look?
image

image

Note: There is no UI to update `expiry`, `name`  ,`description` and `status` . for now it is part of `mc` commands

### CODE REVIEW

File: models/service_accounts.go [hunk 1]
```diff
@@ -24,21 +24,116 @@ package models
...
+	"github.com/go-openapi/errors"
 	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
 )
 
 // ServiceAccounts service accounts
 //
 // swagger:model serviceAccounts
-type ServiceAccounts []string
+type ServiceAccounts []*ServiceAccountsItems0
```
Comment (kaankabalak) [inline models/service_accounts.go:37]: Is the 0 at the end expected for the type here? Or was this previously a placeholder which got generated from the swagger file?

Comment (prakashsvmx) [reply models/service_accounts.go:37]: Yes. It was auto generated by `make swagger-gen`

**Review by cesnietor — APPROVED** [review]
Tested disabling svcacct and status is being reflected.

**Review by jinapurapu — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM, `string[]` change in PolicyDetails.tsx is fine

**Pull request merged**